float32 x # in m/s
float32 y # in m/s
float32 variance_x # uncertainty in the velocity
float32 variance_y # uncertainty in the velocity
